helpers/keychain.ts

```
@@ -0,0 +1,65 @@
+      const [address, label, walletID] = credentials.password.split(',').map(decodeURIComponent);
+      return { address, label, walletID };
+    }
+
+    return null;
+  } catch (error) {
+    console.error('Error retrieving data from keychain:', error);
+    return null;
+  }
+};
+
+export const deleteFromKeychain = async (key = 'walletaddress', useAccessGroup = true): Promise<void | Error> => {
```

Biome

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)